Source/BSN.Commons/Infrastructure/UpdateConfig.cs

```diff
@@ -0,0 +1,75 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using System.Text;
+
+namespace Commons.Infrastructure
+{
+	public class UpdateConfig<T> : IUpdateConfig<T> where T : class
```

Comment:
Please add some API Comment for this class or ref to the IUpdateConfig

with xml c# doc